app/views/shared/_festival_banner.html.haml

```diff
@@ -6,5 +6,5 @@
           = link_to image_tag('festival/codebar-festival-logo-light.png', style: 'max-width: 218px;', alt: 'codebar Festival logo'), 'https://festival.codebar.io/', class: 'border-0'
         .col-md
           %p.text-white.mb-2
-            %strong 6 - 11th March 2023
-          %p.text-white.mb-0 Build a holistic career in tech with us at codebar Festival! Join us on the 6th, 7th, 8th, 9th and 11th of March for 5 days of workshops, talks, panels and networking opportunities. #{link_to 'Checkout out the schedule and RSVP here', 'https://festival.codebar.io', class: 'link-light border-0 text-decoration-underline'}.
+            %strong 4 - 9th March 2024
+          %p.text-white.mb-0 Build a holistic career in tech with us at codebar Festival! Join us on the 4th, 5th, 6th, 7th, 8th and 9th of March for 6 days of workshops, talks, panels and networking opportunities. #{link_to 'Checkout out the schedule and RSVP here', 'https://festival.codebar.io', class: 'link-light border-0 text-decoration-underline'}.
```

Comment:
Currently we aren't running anything on Friday 8th so that day needs to be removed which also means it's 5 days of workshops

Reply:
@KimberleyCook updated!